generator/config/stages.yaml

```diff
@@ -0,0 +1,33 @@
+- name: match
+  type: stage
+  args:
+    - name: query
+      type: expression
+- name: sort
+  type: stage
+  args:
+    # @todo should generate be a map, not a list
+    - name: sortSpecification
+      type: number # should be an enum PHPLIB-1269
```

Comment:
`number` is incorrect: the sort specification also supports [Text Score Metadata Sort](https://www.mongodb.com/docs/manual/reference/operator/aggregation/sort/#std-label-sort-pipeline-metadata)

Reply:
That's handled by [PHPLIB-1269](https://jira.mongodb.org/browse/PHPLIB-1269).

Reply:
See [5bf4753](https://github.com/mongodb/mongo-php-library/pull/1174/commits/5bf47537028684b7fb5697de97ad32b82ca33229)